src/index.js

```diff
@@ -45,11 +46,13 @@ export default function loader(src) {
     const esModule =
       typeof options.esModule !== 'undefined' ? options.esModule : true;
 
+    const encodedData = options.generator
+      ? options.generator(src)
+      : `data:${mimetype || ''};${encoding},${src.toString(encoding)}`;
```

Comment:
Look here https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

`encoding` is optinal, so

`data:text/html,%3Ch1%3EHello%2C%20World!%3C%2Fh1%3E` is valid, but in our case we always insert `;`

Rewrite it to ``data:${mimetype || ''}${encoding? `;${encoding}` : ''},${src.toString(encoding)}`;`

Reply:
I've added `boolean` type for encoding, now we can disable it, if it's enabled it will use the default `base64`